src/components/PostCard/PostCard.test.tsx

```diff
@@ -23,4 +25,49 @@ describe("Given the PostCard component", () => {
       expect(title).toBeInTheDocument();
     });
   });
+
+  describe("When it receives post with a author: 'Macario'", () => {
+    test("Then it should show a text with a author 'Macario'", () => {
+      const expectedAuthor = /macario/i;
+      render(<PostCard post={post} />);
+
+      const author = screen.getByText(expectedAuthor);
+
+      expect(author).toBeInTheDocument();
+    });
+  });
+
+  describe("When it receives post with a date: 1714751590000", () => {
+    test("Then it should show a date with 'Fri May 03 2024'", () => {
+      const expectedDate = "Fri May 03 2024";
+      render(<PostCard post={post} />);
+
+      const date = screen.getByText(expectedDate);
+
+      expect(date).toBeInTheDocument();
+    });
+  });
+
+  describe("When it receives post with a image with a alt: 'una araña'", () => {
```

Comment:
si en el post se llama alternativeText, deberia ser, when it receives a post with an _alternative text_..,

Reply:
cambiado